features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/crypto/identity/IdentityChangeState.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2024 New Vector Ltd.
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ * Please see LICENSE in the repository root for full details.
+ */
+
+package io.element.android.features.messages.impl.crypto.identity
+
+import io.element.android.libraries.matrix.api.encryption.identity.IdentityState
+import io.element.android.libraries.matrix.api.room.RoomMember
+import kotlinx.collections.immutable.ImmutableList
+
+data class IdentityChangeState(
+    val roomMemberIdentityStateChanges: ImmutableList<RoomMemberIdentityStateChange>,
+    val eventSink: (IdentityChangeEvent) -> Unit,
+)
+
+data class RoomMemberIdentityStateChange(
+    val roomMember: RoomMember,
```

Comment:
Could we create a specific data class instead of passing `RoomMember` if we only need a few fields?